src/dmtcpplugin.cpp

```diff
@@ -199,7 +199,9 @@ EXTERNC int dmtcp_set_ckpt_dir(const char* dir)
 EXTERNC const char* dmtcp_get_coord_ckpt_dir(void)
 {
   static string dir;
+  dmtcp_disable_ckpt();
```

Comment:
I believe we should be using the helper macros instead of calling the functions directly.